basics/data-import/pinot-stream-ingestion/amazon-kinesis.md

```diff
@@ -55,9 +61,9 @@ Kinesis supports authentication using the [DefaultCredentialsProviderChain](http
 * Credentials delivered through the Amazon EC2 container service if `AWS_CONTAINER_CREDENTIALS_RELATIVE_URI` environment variable is set and security manager has permission to access the variable,
 * Instance profile credentials delivered through the Amazon EC2 metadata service
 
-You can also specify the accessKey and secretKey using the properties. However, this method is not secure and should be used only for POC setups. You can also specify other aws fields such as AWS\_SESSION\_TOKEN as environment variables and config and it will work.
+You can also specify the `accessKey` and `secretKey` using the properties. However, this method is not secure and should be used only for POC setups. You can also specify other AWS fields such as AWS\_SESSION\_TOKEN as environment variables and config and it will work.
 
 #### Limitations
 
-1. ShardID is of the format "**shardId-000000000001**". We use the numeric part as partitionId. Our partitionId variable is integer. If shardIds grow beyond Integer.MAX\_VALUE, we will overflow
+1. `ShardID` is of the format "**shardId-000000000001**". We use the numeric part as `partitionId`. Our `partitionId` variable is integer. If `shardIds` grow beyond Integer.MAX\_VALUE, we will overflow
```

Comment:
```suggestion
1. `ShardID` is of the format "**shardId-000000000001**". We use the numeric part as `partitionId`. Our `partitionId` variable is integer. If `shardIds` grow beyond `Integer.MAX\_VALUE`, we will overflow.
```

Reply:
what does "we will overflow" mean?--can we clarify?

where can this setting be updated?